app/controllers/application_controller.rb

```diff
@@ -126,4 +126,8 @@ def anonymous_user
   def api_action?
     self.class.to_s.start_with?("Api::")
   end
+
+  def initialize_stripe
+    Stripe.api_key = SiteConfig.stripe_api_key
+  end
```

Comment:
I wonder if this could be extracted in a concern in https://github.com/forem/forem/tree/master/app/controllers/concerns, not blocking though

Reply:
Yeah I think this is ideal, but I think I'll leave this for a follow-on PR to make sure it gets the proper testing etc. At least this is stupid simple by putting it directly in the traffic path.